docs/5.0/preview/database-access.md

```diff
@@ -44,23 +44,82 @@ The following diagram shows an example Database Access setup:
 
 ## Schedule
 
-!!! info
-    Teleport Database Access is under active development.
-    The alpha release will include support for PostgreSQL, including Amazon RDS and Aurora.
+Teleport Database Access is under active development. The alpha release will
+include support for PostgreSQL, including Amazon RDS and Aurora.
 
-[Database Access Release Schedule](./upcoming-releases.md#release-schedule)
+See [release schedule](./upcoming-releases.md#release-schedule).
 
-## Setup
+## Configure PostgreSQL
 
-Let's setup a sample Teleport Database Access deployment.
+### On-Prem PostgreSQL
 
-We will create a Teleport cluster and connect a PostgreSQL-flavored AWS Aurora
-database to it via Teleport database proxy.
+!!! note
+    This section explains how to configure an on-premise instance of PostgreSQL
+    to work with Teleport Database Access. For information about configuring
+    AWS RDS/Aurora see the [section below](#aws-rdsaurora-postgresql).
 
-### Setup Aurora
+#### Create Certificate/Key Pair
 
-Teleport Database Access uses IAM authentication with AWS RDS and Aurora
-databases which can be enabled with the following steps.
+Teleport uses mutual TLS authentication so PostgreSQL server must be configured
+with Teleport's certificate authority and certificate/key pair that Teleport
+can validate.
+
+To create these secrets, use `tctl auth sign` command. Note that it requires a
+running Teleport cluster and [should be run](https://goteleport.com/teleport/docs/architecture/overview/#tctl)
+on the auth server.
+
+```sh
+$ tctl auth sign --format=db --host=db.example.com --out=server --ttl=8760h
+```
+
+Flag descriptions:
+
+* `--format=db`: instructs the command to produce secrets in the format suitable
+  for configuring a database server.
+* `--host=db.example.com`: server name to encode in the certificate, should
```

Comment:
If using AWS does this also need to map to any AWS RDS name/setting?

Reply:
No, generating certificates is only required for self-hosted instances. RDS users don't need to do this at all.